framework/test/mocha/functional/http/post/0.transfer.js

```diff
@@ -26,7 +26,7 @@ const phases = require('../../../common/phases');
 const sendTransactionPromise = require('../../../common/helpers/api')
 	.sendTransactionPromise;
 const randomUtil = require('../../../common/utils/random');
-const errorCodes = require('../../../../../src/modules/chain/helpers/api_codes');
+const apiCodes = require('../../../../../src/modules/http_api/api_codes');
```

Comment:
Prefer `apiErrorCodes`

Reply:
`api_codes.js` file not only contains error codes but also `< 200` codes too. I think it's more consistent to what `api_codes.js` contains to leave it as `apiCodes`. Please, let me know what you think.